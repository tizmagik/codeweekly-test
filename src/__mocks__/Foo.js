import React from 'react';

export default () => {
  console.log('Mocked foo!');
  return <div>Mocked Foo!</div>;
}
