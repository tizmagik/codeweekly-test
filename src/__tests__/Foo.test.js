import React from 'react';
import Foo from '../Foo';
import { shallow } from 'enzyme';

describe('Foo', () => {
  it('Renders', () => {
    expect(shallow(<Foo />)).toMatchSnapshot();
  })
})
