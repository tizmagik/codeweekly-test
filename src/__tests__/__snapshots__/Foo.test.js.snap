exports[`Foo Renders 1`] = `
<div>
  Hello
</div>
`;
